lib/api/stitch_plan.py

```diff
@@ -17,9 +17,17 @@ def get_stitch_plan():
         return dict(colors=[], stitch_blocks=[], commands=[])
 
     metadata = g.extension.get_inkstitch_metadata()
+
+    page_specs = {
+        "width": metadata.document.get('width'),
+        "height": metadata.document.get('height'),
+        "pagecolor": metadata.document[1].get('pagecolor'),
+        "deskcolor": metadata.document[1].get('inkscape:deskcolor')
+    }
+
     collapse_len = metadata['collapse_len_mm']
     min_stitch_len = metadata['min_stitch_len_mm']
     patches = g.extension.elements_to_stitch_groups(g.extension.elements)
-    stitch_plan = stitch_groups_to_stitch_plan(patches, collapse_len=collapse_len, min_stitch_len=min_stitch_len)
+    stitch_plan = stitch_groups_to_stitch_plan(patches, collapse_len=collapse_len, min_stitch_len=min_stitch_len, page_specs=page_specs)
```

Comment:
I'd rather if we didn't wedge the page specs into the stitch plan.  It doesn't feel like they belong there.

How about a separate API call to fetch the page specs?

Reply:
That's a good idea, I'll figure it out.

Reply:
Moved to separate api call